dozer-orchestrator/src/pipeline/source_builder.rs

```diff
@@ -149,6 +149,11 @@ mod tests {
             endpoints: vec![],
             sql: None,
             home_dir: "test".to_string(),
+            cache_max_map_size: Default::default(),
```

Comment:
These defaults are `0` right? Is this what we want?

Reply:
nope, I forgot to use the default config functions, thanks for note this out...
should be fixed now